src/Microsoft.AspNetCore.Mvc.Razor/Resources.resx

```diff
@@ -201,7 +201,7 @@
     <value>A circular layout reference was detected when rendering '{0}'. The layout page '{1}' has already been rendered.</value>
   </data>
   <data name="Compilation_DependencyContextIsNotSpecified" xml:space="preserve">
-    <value>One or more compilation references are missing. Possible causes include a missing '{0}' property under '{1}' in the application's {2}.</value>
+    <value>One or more compilation references are missing. Possible causes include a missing '{0}' property in the application's '{1}' file.</value>
```

Comment:
Spoke to @Eilon and we came up with this:

> One or more compilation references are missing. Ensure that your project is referencing '{0}' and the '{1}' property is not set to false.

{0} - Microsoft.NET.Sdk.Web
{1} - PreserveCompilationContext

that get passed in.


`